ng-dev/release/publish/test/release-notes/generation.spec.ts

```diff
@@ -436,4 +437,37 @@ describe('release notes generation', () => {
 
     expect(await releaseNotes.getCommitCountInReleaseNotes()).toBe(4);
   });
+
+  describe('updates the changelog file', () => {
+    it('prepending the entry', async () => {
+      writeFileSync(`${testTmpDir}/CHANGELOG.md`, '<Previous Changelog Entries>');
+
+      const sandboxRepo = SandboxGitRepo.withInitialCommit(githubConfig)
+        .createTagForHead('startTag')
+        .commit('fix(ng-dev): commit *1', 1);
+
+      const fullSha = sandboxRepo.getShaForCommitId(1, 'long');
+      const shortSha = sandboxRepo.getShaForCommitId(1, 'short');
+
+      const releaseNotes = await ReleaseNotes.forRange(parse('13.0.0'), 'startTag', 'HEAD');
+      await releaseNotes.prependEntryToChangelog();
+
+      const changelog = readFileSync(`${testTmpDir}/CHANGELOG.md`, 'utf8');
+
+      expect(changelog).toBe(
+        dedent`
+      <a name="13.0.0"></a>
+      # 13.0.0 (${buildDateStamp()})
+      ### ng-dev
+      | Commit | Type | Description |
+      | -- | -- | -- |
+      | [${shortSha}](https://github.com/angular/dev-infra-test/commit/${fullSha}) | fix | commit *1 |
```

Comment:
Seems like this should be tested in a dedicated spec. This should just confirm the prepending.

We don't have such a test yet; so that is great! but we should have a dedicated test instead IMO.